importer/helpers/helpers.go

```diff
@@ -124,9 +137,20 @@ func (n *UnixfsNode) SetData(data []byte) {
 	n.ufmt.Data = data
 }
 
+func (n *UnixfsNode) DataSize() uint64 {
```

Comment:
Is there a reason you called this DataSize() and not FileSize()?  FileSize() means the size of the file _if this node was the root_.  In #3314 I added a similar helper method but I called it FileSize() and not DataSize().